src/rules/requireJsdoc.js

```diff
@@ -7,6 +7,12 @@ import getJSDocComment from '../eslint/getJSDocComment';
 const OPTIONS_SCHEMA = {
   additionalProperties: false,
   properties: {
+    contexts: {
```

Comment:
This option overlapped the option `require`?

Reply:
Good catch. Yes, I avoided a breaking change here in order to allow `require-jsdoc` to offer more arbitrary contexts while also phasing in the API used by `require-description` and `match-description`. I can make a breaking change to remove `require` in favor of `contexts` if you and @gajus wish. I think it is a good idea.

Reply:
I am ok either way.

Reply:
Regarding breaking-changes: I am Okay with as many breaking changes as you need. Far bigger preference for clean, unambiguous APIs without legacy remnants over how often a breaking-change is made. This seems like an unpopular opinion in JavaScript land, but in my opinion a greater number of breaking changes over time is better (as it allows more gradual adoption, one version at a time) than one major planned breaking change that batches together many unrelated API changes and requires significant time investment to upgrade.

Reply:
Yes, it is jarring at first when one sees a "major" change and expects it will require a huge number of changes, but if people get to know the repository/package (or note how large its major version already has become), and how it may do frequent though incremental breaking releases, the angst, as well as the incremental effort is less.